executors/src/eoa/worker/confirm.rs

```diff
@@ -108,8 +109,38 @@ impl<C: Chain> EoaExecutorWorker<C> {
                     {
                         tracing::warn!(
                             error = ?e,
-                            "Failed to attempt gas bump for stalled nonce"
+                            "Failed to attempt gas bump for stalled nonce, trying fallback"
                         );
+                        
+                        // Fallback: try to send a no-op transaction
+                        tracing::info!(
+                            nonce = transaction_counts.preconfirmed,
+                            "Gas bump failed, attempting no-op transaction as fallback"
+                        );
+                        if let Ok(noop_tx) = self.send_noop_transaction(transaction_counts.preconfirmed).await {
```

Comment:
Kinda wish this change was isolated in case we need to revert it. Let's go with it but for next time you should break out smaller prs